deploy/crane-namespace.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
```

Comment:
we need to support deploy manager by `kubectl apply -f deploy/crane-manager` ,it's simple to use ,so i suggest to put namespace-creation.yaml in deploy/crane-manager